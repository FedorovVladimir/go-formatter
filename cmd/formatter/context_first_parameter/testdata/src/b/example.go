package a

import "context"

func ctxFirstParameter(a int, ctx context.Context) { // want "context first parameter"

}
