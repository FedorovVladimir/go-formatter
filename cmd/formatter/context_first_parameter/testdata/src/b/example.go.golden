package a

import "context"

func ctxFirstParameter(ctx context.Context, a int) { // want "context first parameter"

}